### ISSUE TITLE
SDL Bump or ImGUI Broke Android Context Controls and Joystick is Anchored

### ISSUE THREAD

**Issue by chriseich21** [CleverRaven/Cataclysm-DDA#78087]
### Describe the bug

Context controls do not display when navigating main menu on boot. 
Virtual joystick is anchored to top left corner (input is less accurate).
Once context controls are displayed, they are not selectable (selects whatever is highlighted on game window)

### Attach save file

N/A

### Steps to reproduce

1. Install cdda-experimental-2024-04-05-2105 or later (still present in 11-03-2024)
2. Observe missing context control bar on main menu when opened
3. Display context control bar (i.e. Options menu > Default Character Name > Cancel Text Input)
4. Attempt to select a context control option (i.e. Tab to go to next Options page)
5. Enter key is input instead of selected control (Default Char Name input reopens)

### Expected behavior

I would expect no regression in context control behavior after #72154 and #72345 integration

### Screenshots

3eb062c Version Not Working Screenshot:
![3eb062c_Not_Working](https://github.com/user-attachments/assets/5f36df79-ddda-4537-9a8a-bf3db9f7f43b)

dd1adff Version Working Screenshot:
![dd1adff_Working](https://github.com/user-attachments/assets/09e96fbf-0ab4-4566-975a-2a44afeac674)

### Versions and configuration

OS: Android 14 64bit
Game Version (Not Working): 3eb062c (cdda-android-x64-2024-04-05-2105)
Game Version (Working): dd1adff (cdda-android-x64-2024-02-26-0341)

**Comment by chriseich21** [comment]
3eb062c Version Joystick Anchored:
![3eb062c_Joystick](https://github.com/user-attachments/assets/e5c1a80d-7686-43e9-a951-404cc62aaa15)

dd1adff Version Joystick Not Anchored:
![dd1adff_Joystick](https://github.com/user-attachments/assets/71ce633f-d019-40f5-9a63-806f6a4c109d)

**Comment by Jungle93** [comment]
I encountered with the same problems on .H release version.I can't play any more.There is some env **context info ：**
Phone：HORNOR 80 GT
OS：Android 12 64bit
Game Version (Not Working): cdda-android-x64-2024-11-23-0334

**Comment by Jungle93** [comment]
> I encountered with the same problems on .H release version.I can't play any more.There is some env **context info ：** Phone：HORNOR 80 GT OS：Android 12 64bit Game Version (Not Working): cdda-android-x64-2024-11-23-0334

Yep.I solve the problem by turn the software rendering to off! Can you give it a try?@chriseich21

**Comment by chriseich21** [comment]
Herbert stable context control menu is working with software rendering disabled. I believe I turned software rendering on to fix something in the past but it seems okay disabled.
